config/kubernetes-sigs/sig-api-machinery/teams.yaml

```diff
@@ -59,27 +59,15 @@ teams:
   controller-tools-admins:
     description: admin access to controller-tools
     members:
-    - DirectXMan12
-    - droot
-    - pwittrock
```

Comment:
These folks have moved to emeritus a while back: https://github.com/kubernetes-sigs/controller-runtime/blob/260b822e9daa53b2ddec007ec48359e38a3ef797/OWNERS_ALIASES#L39-L41